hack/build/cleanup_docker.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+REPO=
```

Comment:
You can replace the whole setTag function with this:

REPO=${DOCKER_REPO:-"localhost"}

That basically means REPO = ${DOCKER_REPO} if not found, use "localhost"

Reply:
> You can replace the whole setTag function with this:
> 
> REPO=${DOCKER_REPO:-"localhost"}
> 
> That basically means REPO = ${DOCKER_REPO} if not found, use "localhost"

Nice! Updated